rules/zoom_operation_rules/zoom_new_meeting_passcode_required_disabled.py

```diff
@@ -0,0 +1,15 @@
+def rule(event):
+    operation_detail = event.get("operation_detail", "<NO_OPS_DETAIL>")
+    operation_flag = "Security  - Require a passcode when scheduling new meetings: from On to Off"
+    return (
+        event.get("action", "<NO_ACTION>") == "Update"
+        and event.get("category_type", "<NO_CATEGORY_TYPE>") == "Account"
+        and operation_flag == operation_detail
+    )
```

Comment:
```suggestion
    return all([
        event.get("action", "<NO_ACTION>") == "Update",
        event.get("category_type", "<NO_CATEGORY_TYPE>") == "Account",
        operation_flag == operation_detail
    ])
```

Reply:
included this change in most recent branch update